plugins/jira/plugin.spec.yaml

```diff
@@ -5,7 +5,6 @@ name: jira
 title: Jira
 vendor: rapid7
 support: rapid7
-cloud_ready: true
```

Comment:
I think that this should be left as is, since we added this here for internal testing: https://github.com/rapid7/insightconnect-plugins/pull/787/files#diff-bd066d441f6a6d9425c2bda33af22e1a451e5c793891bbec54049ba1632fa7e5R10

Reply:
Point taken. Rush to delete came from misunderstanding of subsequent ticket. Tag added back.